lib/page/house_wikidata.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../../lib/popolo_helper.rb'
+
+module Page
+  class HouseWikidata
+    def initialize(slug, house)
+      @house = house
+      @slug = slug
+    end
```

Comment:
This reads quite oddly as the two instance variables look like they're at two different levels of specificity, although they're actually both the same. `slug` here is the *country* slug, and `house` is the *house* slug.